src/bedrock_agentcore/memory/client.py

```diff
@@ -92,6 +92,47 @@ def create_memory(
             logger.error("Failed to create memory: %s", e)
             raise
 
+    def create_or_get_memory_id(
```

Comment:
I think we should return the entire memory object here, we can always extract the id from the returned memory object and rename the function appropriately to `get_or_create_memory`